lib/win32.js

```diff
@@ -53,20 +54,22 @@ exports.prepare = (device) => {
     const deviceId = _.nth(device.match(/PHYSICALDRIVE(\d+)/i), 1);
 
     if (deviceId) {
+
+      // Test whether the device is accessible,
+      // and if not – throw – rejecting the promise
+      fs.accessSync(device, fs.constants.R_OK | fs.constants.W_OK);
+
       return retry(() => {
```

Comment:
Notice that in the case of https://github.com/resin-io/etcher/issues/1127, we have to check the drive exists inside the retry block, otherwise we catch an initial inconsistency, but if the drive gets removed in between two tries, the error still appears.